src/Inquirer/Console/ConsoleUtils.cs

```diff
@@ -19,7 +18,10 @@ public static class ConsoleUtils
                                                                 ConsoleKey.D9 };
         public static bool isDigit(ConsoleKey key)
         {
-            return DigitsKeys.Contains(key);
+            //Update Oct 1, 2020
+            //If the digit is D2 (@ special character) then it will go back one character and the next will replace it
+            //Adding a condition to avoid this and include the @ special character
+            return (key != ConsoleKey.D2) && DigitsKeys.Contains(key);
```

Comment:
If we add here this condition, this method will not represent what it says that it does. Like, if the key is D2 it is a Digit.

I think that the best place to add this bug fix is on `ConsoleObservable:ImplementKeysBehaviours`, that is the method that has the bug.

Reply:
That makes sense. Let me apply the change real quick